rewriter/Minitest.cc

```diff
@@ -420,28 +533,97 @@ ast::ExpressionPtr runSingle(core::MutableContext ctx, bool isClass, ast::Send *
         return constantMover.addConstantsToExpression(send->loc, move(method));
     }
 
+    if (send->numPosArgs() == 0 && send->fun == core::Names::subject()) {
+        auto declLoc = send->funLoc;
+        return ast::MK::SyntheticMethod0(send->loc, declLoc, core::Names::subject(), std::move(block->body));
```

Comment:
Is this missing an `addSigVoid`?

Reply:
I don't think so... `subject` is like `let` in that it returns a non-void type. When I add `addSigVoid` I get 2K type errors like this:
```
packs/compliance/spec/domain/customer_creation_request_manager_spec.rb:404: Cannot call method developer_accepted_tos on void type https://srb.help/7003
     404 |          expect(subject.developer_accepted_tos).to eql(false)
```